apps/cyberstorm-storybook/stories/components/PackageCard.stories.tsx

```diff
@@ -7,9 +7,7 @@ const meta = {
   component: PackageCard,
 } as ComponentMeta<typeof PackageCard>;
 
-const defaultArgs = {
-  packageCardStyle: "default",
-};
+const defaultArgs = {};
```

Comment:
I don't see much point in having an empty object as the defaults.

Reply:
Removed (also removed from Tag)